src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -40,8 +40,6 @@ class Serializer implements SerializerInterface, NormalizerInterface, Denormaliz
     protected $encoder;
     protected $decoder;
     protected $normalizers = array();
-    protected $normalizerCache = array();
-    protected $denormalizerCache = array();
```

Comment:
Oh, didn't see they were protected and not private. Should i just keep them or keep the logic that is filling them to ?